v19.2/show-ranges.md

```diff
@@ -33,245 +25,88 @@ Parameter | Description
 [`table_index_name`](sql-grammar.html#table_index_name) | The name of the index you want range information about.
 [`database_name`](sql-grammar.html#database_name) | The name of the database you want range information about.
 
-## Examples
-
-The following examples use MovR, a fictional vehicle-sharing application, to demonstrate CockroachDB SQL statements. For more information about the MovR example application and dataset, see [MovR: A Global Vehicle-sharing App](movr.html).
-
-### Show manually-split ranges
-
-You can use the [`SPLIT AT`](split-at.html) statement to manually split table ranges based on primary and secondary index values.
+## Response
 
-#### Setup
+The following fields are returned for each partition:
 
-To follow along with the example statements, run [`cockroach demo`](cockroach-demo.html) to start a temporary, in-memory cluster with the `movr` dataset preloaded:
+Field | Description
+------|------------
+`table_name` | The name of the table.
+`start_key` | The start key for the range.
+`end_key` | The end key for the range.
+`range_id` | The range ID.
+`range_size_mb` | The size of the range.
+`lease_holder` | The node that contains the range's [leaseholder](architecture/overview.html#glossary).
+`lease_holder_locality` | The [locality](start-a-node.html#locality) of the leaseholder.
+`replicas` | The nodes that contain the range [replicas](architecture/overview.html#glossary).
+`replica_localities` | The [locality](start-a-node.html#locality) of the range.
 
-{% include copy-clipboard.html %}
```

Comment:
I guess the idea is that if you use movr we don't need all this, right?  Big win if so!